addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -29,8 +36,25 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
     init: function (parent, state, params) {
         this._super(...arguments);
         this.breadcrumbs = params.breadcrumbs;
+        this.chatterFields = {
+            // `knowledge.article` has `mail.activity.mixin`
+            hasActivities: this.state.model === 'knowledge.article',
+            hasFollowers: true,
```

Comment:
Side question: are followers managed somehow (aka: if someone that has no access is added as follower, tant pis ?)

Reply:
I'll investigate ;p